docs/contributing/guidelines/ci.md

```diff
@@ -24,7 +24,7 @@ Travis configuration is located in the [.travis.yml](https://github.com/ARMmbed/
    - Features/netsocket.
 - **travis-ci/events** - Checks that Mbed OS compiles and run events tests.
 - **travis-ci/gitattributestest** - Checks there are no changes after clone. This checks that `.gitattributes` is used correctly.
-- **travis-ci/licence_check** - Checks there is no GPL license text in the code.
+- **travis-ci/licence_check** - Checks only a permissive license in the files, including SPDX identifier.
```

Comment:
I think i missed a bit:

Checks that there is only a permissive license in the files,

Reply:
fixed